config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Service.java

```diff
@@ -6,7 +6,9 @@
 
 package com.ca.apim.gateway.cagatewayconfig.beans;
 
+import com.ca.apim.gateway.cagatewayconfig.config.loader.FolderLoaderUtils;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.ca.apim.gateway.cagatewayconfig.config.loader.FolderLoaderUtils'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=128&issues=AWcPsqrZd6oxh3R491C8&open=AWcPsqrZd6oxh3R491C8)

[ISSUE_KEY]: AWcPsqrZd6oxh3R491C8